File: csharp/SlimDownYourAggregates.Tests/Slimmed/Entities/Format.cs
namespace SlimDownYourAggregates.Tests.Slimmed.Entities;

public class Format
{
    public Format(string formatType, int totalCopies, int soldCopies)
    {
        FormatType = formatType;
        TotalCopies = totalCopies;
        SoldCopies = soldCopies;
    }

    public string FormatType { get; }
    public int TotalCopies { get; }
    public int SoldCopies { get; }
}

File: java/src/main/java/io/eventdriven/slimdownaggregates/shorter/original/entities/Format.java
package io.eventdriven.slimdownaggregates.shorter.original.entities;

public class Format {
  private final String formatType;
  private final int totalCopies;
  private final int soldCopies;

  public Format(String formatType, int totalCopies, int soldCopies)
  {
    this.formatType = formatType;
    this.totalCopies = totalCopies;
    this.soldCopies = soldCopies;
  }

  public String getFormatType() {
    return formatType;
  }

  public int getTotalCopies() {
    return totalCopies;
  }

  public int getSoldCopies() {
    return soldCopies;
  }
}
